fit/wa-reference/src/main/resources/log4j2.xml

```diff
@@ -63,7 +63,11 @@ under the License.
       <appender-ref ref="main"/>
     </asyncLogger>
 
-    <asyncLogger name="org.apache.cxf" additivity="false" level="ERROR">
+    <asyncLogger name="org.apache.cxf" additivity="false" level="DEBUG">
+      <appender-ref ref="main"/>
+    </asyncLogger>
+
+    <asyncLogger name="org.springframework.security" additivity="false" level="DEBUG">
```

Comment:
Please revert, as this file is taken by archetype as reference for new projects.